### ISSUE TITLE
problem with <lem type="omitted_elsewhere">

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/tfd-nusantara-philology#162]
I am not sure what the source of the following problem is. It may be related to #132.

In Svayambhu §109, we have this string of code:

`<app><lem type="omitted_elsewhere">tan <app><lem wit="#M">hana</lem><rdg wit="#K"><gap reason="omitted"/></rdg></app>  <app><lem type="norm">pakənanikaṅ</lem><rdg wit="#K">pakaḍananiṁ</rdg><rdg wit="#M">pakənaniṅkaṅ</rdg></app> brāhmaṇa,</lem><witDetail type="retained" wit="#K #M"/><note type="altLem">tan hana ... brāhmaṇa</note><rdg wit="#L" cause="eye-skip"><gap reason="omitted"/></rdg></app>` 

The contents of the substring `<app><lem type="norm">pakənanikaṅ</lem><rdg wit="#K">pakaḍananiṁ</rdg><rdg wit="#M">pakənaniṅkaṅ</rdg></app>` is correctly displayed on its own in app780 but its two `<rdg>` also appear — without this being intended or desired — in  app778. The indication "om. K" in app778 is contradictory to the indication "transmitted in K M" (generated by `<witDetail type="retained" wit="#K #M"/>`) and seems to replicate the indication for #K from app779.

<img width="1398" alt="Capture d’écran 2024-12-20 à 14 17 11" src="https://github.com/user-attachments/assets/5336df88-ad61-4036-a624-248c4ad38f05" />
<img width="1417" alt="Capture d’écran 2024-12-20 à 14 29 53" src="https://github.com/user-attachments/assets/2bf5e0aa-99b2-453b-964d-3d0bf3c15a62" />
<img width="1340" alt="Capture d’écran 2024-12-20 à 14 21 57" src="https://github.com/user-attachments/assets/99223299-7ae2-41f4-8eaa-c26ef92df974" />

In short, here and probably also in other cases, the display of apparatus entries that fall within a larger gap in one of the witnesses is quite messy and needs to be sorted out. **The sooner the better, because we need a clean display in order to do quality control on our editions.**

**Comment by arlogriffiths** [comment]
Another problem with "omitted_elsewhere" surfaces at the start of the Sarvavajrodaya edition, where the expression "transmitted in" is left without siglum in the display:

<img width="1623" alt="Capture d’écran 2025-01-07 à 09 19 26" src="https://github.com/user-attachments/assets/d2862ffe-b365-4711-baaf-e368f6b9b83c" />

As per the remarks above, we also don't want to see the middle part "cintanīyatyādi- , om. EdMSK" in this apparatus entry.

**Comment by ajaniak** [comment]
It should be applied at some point in the afternoon.

**Comment by arlogriffiths** [comment]
now we have unwanted italics on EdT :) no hurry to fix this, of course.

<img width="311" alt="Image" src="https://github.com/user-attachments/assets/afec6edf-086d-4160-ac08-2c8dedc55d1e" />

**Comment by arlogriffiths** [comment]
not yet online... 

the above example by the way also fall under the cases for which the usage of color/italics should be rethough. see my other comment of this morning.

I asm not sure what all the variables are so I have difficulty giving concrete recommendations.

**Comment by ajaniak** [comment]
I have forced a run on tfd-sanskrit, it is online now.
